README.md

```diff
@@ -1,8 +1,8 @@
-# portfolio-setup
+# Responsive Portfolio
```

Comment:
- [ ] [OPTIONAL]: You should consider editing this heading to something more personal. For instance "_Your name's_ portfolio"

Reply:
According to the requirement, your report should have a list of everything that was checked, as if it was a checklist, having it "no changes required" for everything is not enough.

Reply:
ok,